torch/nn/utils/rnn.py

```diff
@@ -435,7 +435,7 @@ def unpad_sequence(
     unpadded_sequences = []
 
     if not batch_first:
-        padded_sequences.transpose_(0, 1)
+        padded_sequences = padded_sequences.transpose(0, 1)
```

Comment:
Would it make sense to make the in-place behavior optional (even if off by default) as I assume this incurs some memory overhead?

Reply:
Right. I added an ``inplace`` argument.